### PR TITLE
Cleanup processing and security & privacy

### PR DESCRIPTION
**Pull request by igrigorik**
This should, I believe, address the discussions in:
- https://github.com/w3c/beacon/issues/10
- https://github.com/w3c/beacon/issues/15
- https://github.com/w3c/beacon/issues/16

Preview: https://rawgit.com/w3c/beacon/cleanup/index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -372,17 +371,16 @@
...
         allow all such requests to run to completion without blocking other
         time-critical and high-priority work.</li>
-        <li>The user agent MUST NOT skip or throttle processing of beacon
-        requests, as it may contain critical application state, events, and
-        analytics data.</li>
-        <li>The user agent MUST process the response headers, and if provided,
-        MUST ignore any entity bodies in the response.</li>
       </ul>
+
+      <div class="note">Beacon API does not provide a response callback. The server should not provide a response body for such requests, and the client may terminate responses that contain a body - e.g. cancel the HTTP/2 stream, or abort the HTTP/1 connection.</div>
```
Comment (annevk) [inline index.html]: You're using normative terms "should not" in a note. That's wrong.

File: index.html [hunk 2]
```diff
@@ -504,60 +499,26 @@ <h4 id="return-values">Return Value</h4>
...
+    <section id="privacy" class="informative">
       <h2>Privacy and Security</h2>
-      <p>The <code>sendBeacon</code> interface provides an asynchronous and
-      non-blocking mechanism for delivery of data. It is designed to eliminate
-      the need for the widespread use of expensive anti-patterns that block
-      user interaction, and to enable more efficient scheduling and use of
-      system resources. The same functionality is possible today, albeit with
-      high performance costs and degraged user experience.</p>
-      <p><code>sendBeacon</code> does not add new privacy considerations. However, implementers should take the following precautions:</p>
+      <p>The <code>sendBeacon</code> interface provides an asynchronous and non-blocking mechanism for delivery of data. This API may be used to:</p>
```
Comment (annevk) [inline index.html]: You probably want to use "can be" here since "may" is a normative term and makes little sense to use here.

**Comment by igrigorik** [comment]
@annevk thanks, should be fixed now.

**Comment by toddreifsteck** [comment]
LGTM

**Pull request merged**